biom/table.py

```diff
@@ -522,6 +522,39 @@ def matrix_data(self):
         """The sparse matrix object"""
         return self._data
 
+    def to_pandas(self):
+        """
+        Returns
+        ----------
+        counts : pd.SparseDataFrame
+            Pandas representation of counts
+        sample_metadata : pd.DataFrame
+            Pandas representation of the sample metadata
+        observation_metadata : pd.DataFrame
+            Pandas representation of the observation metadata
+
+        Note
+        ----
+        This can only handle
+        """
+        m = self.matrix_data
+        data = [pd.SparseSeries(m[i].toarray().ravel()) for i in np.arange(m.shape[0])]
```

Comment:
also, why not just `for row in self.matrix_data`? i dont think the explicit index is necessary